kin-devplatform-sdk/src/main/java/kin/devplatform/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -274,18 +283,15 @@ public void onEvent(kin.core.Balance data) {
 	}
 
 	@Override
-	public void removeBalanceObserver(@NonNull Observer<Balance> observer) {
+	public void removeBalanceObserver(@NonNull Observer<Balance> observer, boolean stopSSE) {
 		Logger.log(new Log().withTag(TAG).text("removeBalanceObserver"));
 		balance.removeObserver(observer);
+		if (stopSSE) {
```

Comment:
Just for my knowledge, why don't we decrease the count if we remove the observer? why is it an option?

Reply:
because it's important just for releasing SSE connection when there are no listeners for sse, see my first comment.